allennlp/common/file_utils.py

```diff
@@ -46,15 +46,15 @@
     )
 
 
-def url_to_filename(url: str, etag: str = None) -> str:
+def resource_to_filename(resource: str, etag: str = None) -> str:
```

Comment:
I was a little hesitant to change the name of this function because it's technically not a "private" function, but I don't believe it was ever intended to be used outside of this module. Maybe we should give it a leading underscore to make it clear that it's supposed to be "private".

Reply:
Yes, underscore is the way to go.